Model/DutyCalculator.php

```diff
@@ -132,10 +132,28 @@ public function getDutyandTax($cartId, $shippingCharge, $shippingMethodCode, $sh
                 return $this->response;
             }
             $accessToken = $this->getDhlAccessToken();
-           
+
+            // Enable lines with logger function to turn on logging for debugging.
+            // $this->_logger->debug('---------------- getDutyandTax - START OF REQUEST----------------');
```

Comment:
It would be a better practice for us not to comment out logging statement but to change log level based on which logging would be on or off. Thanks.

Reply:
I agree with your statement but my lines are much more verbose than would ever be needed. I would prefer to keep them commented out for now so they can be easily found and remove them completely when we have a better working version of the project. I literally dump every variable out and create a ton of noise in the logs that were necessary to figure out how things work.

Reply:
If the team prefers to keep it this way for now then I am okay with that Ryan.

* That said in general this type of things keep accumulating over time which makes me hesitant to ignore this. 

Thanks.